viewer/scrollview.cpp

```diff
@@ -115,7 +114,7 @@ void* ScrollView::MessageReceiver(void* a) {
 
     if (cur->window != NULL) {
       cur->parameter = new char[strlen(p) + 1];
-      strncpy(cur->parameter, p, strlen(p) + 1);
+      strcpy(cur->parameter, p);
       if (strlen(p) > 0) {  // remove the last \n
         cur->parameter[strlen(p)] = '\0';
```

Comment:
That code looks strange as it does not do what the comment claims. It writes a 0 to a byte which is already 0.